public/tools/rmt-client-setup

```diff
@@ -40,6 +40,99 @@ EOT
 exit 1
 }
 
+function http_get_signing_keys_index_html()
+{
+    local index_html_url=`echo "$REGURL" | awk -F/ '{print "https://" $3 "/repo/keys/"}'`
+
+    $CURL --insecure --fail --output $TMPDIR/index.html $index_html_url
+
+    return $?
+}
+
+function parse_index_html()
+{
+    KEY_FILES=()
+
+    while read line; do
+        # This is no substitute for a HTML parser, and probably only works on the index.html generated by RMT
+        KEY_FILES+=(`echo $line | sed -n "/href/ s/.*href=['\"]\([^'\"]*\)['\"].*/\1/gp"`)
+    done < $TMPDIR/index.html
+}
+
+function http_get_key_file()
+{
+    local key_file_url=`echo "$REGURL/$1" | awk -F/ '{print "https://" $3 "/repo/keys/" $4}'`
+
+    $CURL --silent --fail --output "$TMPDIR/$1" $key_file_url
+
+    return $?
+}
+
+function import_key
+{
+    if [ "$1" == "res-signingkeys.key" -a $ISRES -eq 0 ]; then
+        # this is no RES system, so we do not need this key
```

Comment:
This definitely should be dropped. :grimacing:

Reply:
Is there a chance that customers may copy custom keys into their RMT repo/keys when upgrading from SMT, bringing this key with them and then adding it to their new systems. If so wouldn't it make sense to leave this filter in?

Reply:
Ah, it's skipping the keys.
Yes, you are right -- it is possible to have those keys after migrating from SLE11.
But in case of this script, `$ISRES` is always `0`, might make sense to remove it and do a check just based on the file name, then.

Reply:
Ok